aws/templates/id/id_apigateway.ftl

```diff
@@ -146,6 +146,7 @@
 
     [#local id = formatAPIGatewayId(core.Tier, core.Component, occurrence)]
     [#local name = formatComponentFullName(core.Tier, core.Component, occurrence)]
+    [#local docsBucketId = formatS3Id(core.Id, "docs")]
```

Comment:
Should be included in the resources as well? And application_gateway changed to use the resource id?